src/tools/rbd/action/MirrorPool.cc

```diff
@@ -1407,6 +1444,20 @@ int execute_info(const po::variables_map &vm,
     }
   }
 
+  if (mirror_mode != RBD_MIRROR_MODE_DISABLED) {
+    r = rbd.mirror_uuid_get(io_ctx, &mirror_uuid);
+    if (r < 0) {
+      return r;
+    }
+    r = rbd.mirror_remote_namespace_get(io_ctx, &remote_namespace);
+    if (r < 0 && r != -EOPNOTSUPP && r != -ENOENT ) {
+      return r;
+    }
+    if (r < 0) {
+      remote_namespace = namespace_name;
```

Comment:
With `mirror_remote_namespace_get()` now squashing `EOPNOTSUPP` and `ENOENT`, this `if` statement should be unreachable and is to be removed.  (I also edited the comment attached to a line a few lines above -- it used to be a formatting nit but is a code change now.)